examples/cert-manager/cert-manager.cue

```diff
@@ -0,0 +1,49 @@
+package polly
+
+import "github.com/pollypkg/polly/schema:pollyschema"
+
+// Enforce that the emit value of this file unifies with the Polly schema
+pollyschema.PollyPackage
+
+header: {
+    name: "cert-manager"
+    uri:  "github.com/pollypkg/polly/examples/cert-manager"
+    params: {
+        certManagerCertExpiryDays: int | *21,
+        certManagerJobLabel: string | *"cert-manager",
+        // TODO Runbooks are an open question themselves, and not really sure how
+        // we'd even think about the interpolation-inside-interpolation here
+        // certManagerRunbookURLPattern: "https://gitlab.com/uneeq-oss/cert-manager-mixin/-/blob/master/RUNBOOK.md#%s",
+        grafanaExternalUrl: string,
+    }
+}
+
+prometheusAlerts: v0: {
+    CertManagerCertExpirySoon: {
```

Comment:
Maps! YISSSS! :tada: :tada: 

Finally I will be able to stop iterating over the whole thing just to select one alert ^^